doc/source/ray-overview/installation.rst

```diff
@@ -28,6 +31,32 @@ To install Ray libraries:
   pip install -U "ray[rllib]"  # installs Ray + dependencies for Ray RLlib
   pip install -U "ray[serve]"  # installs Ray + dependencies for Ray Serve
 
+From conda-forge
```

Comment:
We already have a section down below called `Installing Ray with Anaconda`.

Reply:
Oy, I missed that, sorry. I will redo this to improve that documentation since it is not really Anaconda but the conda-forge open source channel that provides ray. Anaconda provides the hosting, but people need to use the `-c conda-forge` channel.

Reply:
sg!